identity/pool.go

```diff
@@ -123,13 +132,13 @@ func TestPool(p Pool) func(t *testing.T) {
 		})
 
 		t.Run("case=create and keep set values", func(t *testing.T) {
-			expected := passwordIdentity("file://./stub/identity-2.schema.json", "id-2")
+			expected := passwordIdentity(secondSchema.ID, "id-2")
 			require.NoError(t, p.CreateIdentity(context.Background(), expected))
 			createdIDs = append(createdIDs, expected.ID)
 
 			actual, err := p.GetIdentity(context.Background(), expected.ID)
 			require.NoError(t, err)
-			assert.Equal(t, "file://./stub/identity-2.schema.json", actual.TraitsSchemaURL)
+			assert.Regexp(t, "^http://mock-server.com/schemas/*", actual.TraitsSchemaURL)
```

Comment:
Probably check for the `TraitsSchemaID` as well.

And test the concrete value.